src/ec/packages/ec-component-footer-harmonised/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@ecl-twig/ec-component-footer-harmonised",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "2.12.0",
+  "description": "ECL Twig - EC Footer harmonised",
+  "dependencies": {
+    "@ecl-twig/ec-component-link": "2.11.2"
+  },
+  "devDependencies": {
+    "@ecl/ec-component-footer-harmonised": "2.12.0",
```

Comment:
i think you should use 2.14.0 and also define the component as such, i believe there are small differences in the spec files and anyway since we know we are going for 2.14 is safer to use all the available resources and dependencies from the most updated one.. At least i noticed that https://ec.europa.eu/info/index_en this "href" is present in ecl but not in our dependencies  ;)

Reply:
done ;)